packages/dom/test/serialize-dom.test.js

```diff
@@ -68,6 +78,89 @@ describe('serializeDOM', () => {
       expect(html).not.toMatch('<template shadowroot');
       expect(html).not.toMatch('Hey Percy!');
     });
+
+    it('renders single nested ', () => {
+      if (!navigator.userAgent.toLowerCase().includes('chrome')) {
```

Comment:
```suggestion
      if (!navigator.userAgent.toLowerCase().includes('chrome')) {
```

There is a helper `getTestBrowser` that can be used.